File: problemas/1000/1013.cs
using System;

namespace uri1013
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            int a = int.Parse(entrada[0]);
            int b = int.Parse(entrada[1]);
            int c = int.Parse(entrada[2]);
            int d = (a + b + Math.Abs(a - b)) / 2;
            int maior = (c + d + Math.Abs(c - d)) /2;
            Console.WriteLine($"{maior} eh o maior");
        }
    }
}

File: problemas/1000/1013.php
<?php

$entrada = explode(' ', readline());
$a = (int)$entrada[0];
$b = (int)$entrada[1];
$c = (int)$entrada[2];
$d = ($a + $b + abs($a - $b)) / 2;
$maior = ($c + $d + abs($c - $d)) / 2;
echo "{$maior} eh o maior\n";